src/states/elder/mod.rs

```diff
@@ -2087,6 +2067,16 @@ impl Approved for Elder {
         Ok(())
     }
 
+    fn handle_ack_message_event(&mut self, sec_info: SectionInfo) -> Result<(), RoutingError> {
+        self.chain
+            .update_their_knowledge(*sec_info.prefix(), *sec_info.version());
+        #[cfg(feature = "mock_base")]
+        {
+            self.consensused_on_ack_message = true;
```

Comment:
We should not need `consensused_on_ack_message` anymore, we have their_knowledge to use for testing